Application/EdFi.Ods.AdminApi/Features/ClaimSets/ClaimSetModel.cs

```diff
@@ -28,9 +29,10 @@ public class ClaimSetDetailsModel : ClaimSetModel
 [SwaggerSchema(Title = "ResourceClaim")]
```

Comment:
We can use a base class similar to AuthorizationStrategyModel and AuthorizationStrategyClaimSetModel previously implemented.

Reply:
I'm not sure if I understand your suggestion, but this class was renamed to avoid confusion